lib/kitchen/directions/bake_chapter_review_exercises/v1.rb

```diff
@@ -32,7 +32,13 @@ def bake(chapter:, metadata_source:, append_to:, klass:)
 
       @content = exercise_clipboard.paste
 
-      append_to.append(child: render(file: 'review_exercises.xhtml.erb'))
+      append_to_element = append_to || chapter
+
+      if append_to_element == append_to
+        append_to_element.append(child: render(file: 'child_review_exercises.xhtml.erb'))
+      elsif append_to_element == chapter
+        append_to_element.append(child: render(file: 'direct_review_exercises.xhtml.erb'))
```

Comment:
Don't forget to write specs for this change!

Reply:
After applying solution suggested by Kendra there is no need for new specs. All tests passed.